#
# Copyright (c) 2013, 2015, Oracle and/or its affiliates. All rights reserved.
#
# This patch fixes:
# Bug 16976497 - [XKB] layout at does not have "legacy" variant.
#
--- a/symbols/at	Fri Jun 14 11:38:14 2013
+++ b/symbols/at	Tue Jun 18 14:34:59 2013
@@ -21,3 +21,11 @@
 
     name[Group1]= "German (Austria, Macintosh)";
 };
+
+partial alphanumeric_keys
+xkb_symbols "legacy" {
+
+    include "de(legacy)"
+
+    name[Group1]="German (legacy)";
+};
